Bio/NaiveBayes.py

```diff
@@ -21,7 +21,8 @@
 
 Functions:
     - train     - Train a new naive Bayes classifier.
-    - calculate - Calculate the probabilities of each class, given an observation.
+    - calculate - Calculate the probabilities of each class,
+                  given an observation.
```

Comment:
This looks like the final inconsistent bullet point indentation - thanks for your patience on this.